docs/Product-Overview.md

```diff
@@ -0,0 +1,72 @@
+# Product Overview
+
+## OpenTripPlanner project
+
+OpenTripPlanner is a group of open source software applications that help individuals and organizations 
+identify and digitize multimodal trip plans based on OpenStreetMap (OSM) and other standardized data 
+sources (e.g. GTFS, GBFS, NeTEx).
+
+OpenTripPlanner is a community of dozens of individuals and organizations that work collaboratively to 
```

Comment:
The first two paragraphs start with "OpenTripPlanner is a group of... software" and "OpenTripPlanner is a community" which feels contradictory. Suggestion: drop "OpenTripPlanner is" from the second paragraph: "A community of dozens... work on OpenTripPlanner..."

It's understandable if there's a desire to redefine OpenTripPlanner as the community rather than focus on the software, but this will require a bigger overhaul in the messaging.